SPECS-CROSS/cross-gcc/cross-gcc.spec

```diff
@@ -143,6 +144,13 @@ install -D %{_cross_name}-file_manifest.txt %{buildroot}/%{_cross_prefix}/%{_cro
 # %%post   -p /sbin/ldconfig
 # %%postun -p /sbin/ldconfig
 
+%post 
+
+# Create a symlink from sysroot/usr/include to sysroot/include
+# The GCC toolchain will look for header files under sysroot/usr/include
+mkdir -p %{_cross_sysroot}/usr
```

Comment:
Is it possible to add this symlink file in `%install` and include it in the rpm %files? This would make RPM track the file explicitly after install.

Reply:
Thanks for the suggestion. I moved the symlink creation to the %install section and verified that the symlink still works after the package is installed.